### ISSUE TITLE
[Fixed in GH-6052] safari firefox: Missing Visual Focus on Sign In Form

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4311]
![Screen Shot 2019-08-29 at 7 13 13 AM](https://user-images.githubusercontent.com/15113219/63948944-9bfc4200-ca47-11e9-8f01-ac2c26721d89.png)

Input fields and buttons are missing visual focus

**Comment by Zeyber** [comment]
Can see the focus throughout the form. BUttons are difficult to see focus (raised an issue at #4991)

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 12 57 32 PM](https://user-images.githubusercontent.com/15113219/73562636-8392b800-4429-11ea-9dc2-566209fb2616.png)
![Screen Shot 2020-01-31 at 12 57 44 PM](https://user-images.githubusercontent.com/15113219/73562644-88f00280-4429-11ea-92cc-4057359bd974.png)
![Screen Shot 2020-01-31 at 12 58 05 PM](https://user-images.githubusercontent.com/15113219/73562653-8db4b680-4429-11ea-93a4-71dd7729997e.png)
![Screen Shot 2020-01-31 at 12 57 56 PM](https://user-images.githubusercontent.com/15113219/73562654-8db4b680-4429-11ea-816f-ee06e77d4174.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome
- Firefox
- Safari
